src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PXCodeFixProvider.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using System.Collections.Generic;
+using Acuminator.Utilities.Common;
+
+namespace Acuminator.Analyzers.StaticAnalysis
+{
+	[Shared]
+	[ExportCodeFixProvider(LanguageNames.CSharp)]
+	public class PXCodeFixProvider: CodeFixProvider
```

Comment:
please use a sensible name which will indicate that this is a diagnostic suppression via comment code fix